### ISSUE TITLE
[Keyboard Navigation- Make Code - Change Value]: Heading hierarchy is not defined sequentially under the 'Change value' page.

### ISSUE THREAD

**Issue by monisverma** [microsoft/pxt-microbit#5470]
### **User impact:**
Screen reader dependent users will not understand the page structure properly and they will not be aware of the sections/contents present on the 'Change value' page if the standard heading hierarchy i.e. h1>h2>h3>h4>h5>h6 is not defined sequentially.

### **Note:**
User credentials should NOT be included in the bug.

### **Repro Steps:**

1. Open the URL https://arcade.makecode.com/blocks/variables/change
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify whether the heading hierarchy is sequentially defined under the 'Change value' page or not.

### **Actual Result:**
**Issue:** Heading hierarchy is not defined sequentially under the 'Change value' page.
**Observation:** Headings defined such as "Change value(h1)>>Change a declared variable(h3) >See also
(H2) which is not the correct order. H3 heading is directly defined after h1.

### **Expected Result:**
Solution: Headings should be defined in the correct heading hierarchy i.e. H1>H2>H3>H4>H5>H6.

### **ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:**
https://www.w3.org/WAI/tutorials/page-structure/headings/ 

### **UIA/DOM details available:** 
yes

### **MAS Reference:**
[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)

### **Reference Links:** 

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier. 
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”. 
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### **Test Environment:**
**OS:** Windows 11 22H2 (OS Build:22621.3085).
**Browser:** Chrome Version 121.0.6167.86 (Official Build) (64-bit)
**URL:** https://arcade.makecode.com/blocks/variables/change
**Tools:** Developer tool

### **Attachment:**
<img width="953" alt="image" src="https://github.com/microsoft/pxt-microbit/assets/158456802/76cd934c-f9d9-4b63-b772-ee05c423e3e9">
<img width="953" alt="image" src="https://github.com/microsoft/pxt-microbit/assets/158456802/24aef0af-0eb5-4171-8cf9-29dd79e4f48a">

**Comment by monisverma** [comment]
#Accessibility#A11yMAS;#A11ySev3;#E2E-HCL-MakeCode-Jan2024;#MAS1.3.1;#HCL;#Win11-Chrome;#benchmark
